ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/questionnaire-completed/questionnaire-completed.component.html

```diff
@@ -3,12 +3,116 @@
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYou.answer}">
+          Is there anything that could affect your ability to take part in a video hearing?
+        </td>
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYou.answer}" style="text-align:right;">
+          <strong>{{ model.aboutYou.answer | appBooleanHumanView }}</strong></td>
+      </tr>
+      <tr *ngIf="model.aboutYou.answer">
+        <td class="govuk-table__cell" colspan="2">
+          <span class="govuk-caption-m">{{model.aboutYou.notes}}</span>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">About your location</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
+          Will you have access to a quiet, private room where you can access the internet?
+        </td>
+        <td class="govuk-table__cell" style="text-align:right;">
+          <strong>{{ model.room | appBooleanHumanView }}</strong></td>
+      </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">About your client</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYourClient.answer}">
+          Is there anything that could affect your client's ability to take part in a video hearing?
+        </td>
+        <td [ngClass]="{'govuk-table__cell': !model.aboutYourClient.answer}" style="text-align:right;">
+          <strong>{{ model.aboutYourClient.answer | appBooleanHumanView }}</strong></td>
+      </tr>
+      <tr *ngIf="model.aboutYourClient.answer">
+        <td class="govuk-table__cell" colspan="2">
+          <span class="govuk-caption-m">{{model.aboutYourClient.notes}}</span>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">About the hearing</h2>
+    <table class="govuk-table" style="width:100%;">
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td [ngClass]="{'govuk-table__cell': !model.hearingSuitability.answer}">
+          Based on your knowledge of this case and your client, do you believe there is anything that makes this hearing unsuitable for video?
+        </td>
+        <td [ngClass]="{'govuk-table__cell': !model.hearingSuitability.answer}" style="text-align:right;">
+          <strong>{{ model.hearingSuitability.answer | appBooleanHumanView }}</strong></td>
+      </tr>
+      <tr *ngIf="model.hearingSuitability.answer">
+        <td class="govuk-table__cell" colspan="2">
+          <span class="govuk-caption-m">{{model.hearingSuitability.notes}}</span>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    <h2 class="govuk-heading-m">About your equipment</h2>
```

Comment:
This could be broken out to subcomponents to reduce duplication of styling and maybe even make it more readable. **but** that might be completely overkill 😄 💲 

I'm thinking something like:

```
<app-answer-summary-group title="About your equipment">
  <app-summary-answer [value]="model.computer || appYesNo">Do you have access to a ....</app-summary-answer>
  <app-summary-answer [value]="model.camera || appYesNo">Does your computer have a camera microphone?</app-summary-answer>
</app-answer-summary-group>
```

Reply:
Looking at the current code it looks pretty clean as is, feels like breaking it apart wouldn't add much readability so better leave it as is